ozaria/site/views/play/level/PlayLevelView.js

```diff
@@ -1127,6 +1127,7 @@ class PlayLevelView extends RootView {
       : VictoryModal
     if (this.isCourseMode() || me.isSessionless()) {
       ModalClass = CourseVictoryModal
+      options.capstoneStage = utils.getQueryVariable('capstoneStage')
```

Comment:
Does it make sense to convert the stage to a number outside of CourseVictoryModal and have the modal treat it as a number or undefined?

Reply:
That is actually a good idea to convert it the first time it is used. Thanks! :)